identity-credential/src/presentation/presentation.rs

```diff
@@ -10,19 +10,22 @@ use identity_core::common::Object;
 use identity_core::common::OneOrMany;
 use identity_core::common::Url;
 use identity_core::convert::ToJson;
+use identity_core::crypto::SetSignature;
+use identity_core::crypto::Signature;
+use identity_core::crypto::TrySignature;
+use identity_core::crypto::TrySignatureMut;
 use serde::Serialize;
 
 use crate::credential::Credential;
 use crate::credential::Policy;
 use crate::credential::Refresh;
-use crate::credential::VerifiableCredential;
 use crate::error::Error;
 use crate::error::Result;
 use crate::presentation::PresentationBuilder;
 
-/// A `Presentation` represents a bundle of one or more `VerifiableCredential`s.
+/// A `Presentation` represents a bundle of one or more `Credential`s.
```

Comment:
This was an existing issue but this crate is never touched so might as well do now 😄 . The documentation can link directly to other items by using ```[`SomeStruct`]``` but this was incorrectly using ``` `SomeStruct` ```.

The [sign function](https://github.com/iotaledger/identity.rs/pull/170/files#diff-f866bde21b7bc14af5faca62954b5990f570cf2dafb67ae1853b3025c316bf81R176) has an example of this

Reply:
Ah thought this was another convention since I thought I only saw it in places were it was self-referential, where it makes sense to not link..

I will adjust this where I can find this pattern

Reply:
Not a convention but just written before docs became a bigger focus - the self-referential ones are **also** not needed so those could be updated as you encounter them.